Kentico.Kontent.Management.Tests/Unit/Modules/UrlBuilder/AssetRenditionTests.cs

```diff
@@ -0,0 +1,89 @@
+﻿using FluentAssertions;
+using Kentico.Kontent.Management.Models.AssetRenditions;
+using Kentico.Kontent.Management.Models.Shared;
+using System;
+using Xunit;
+
+namespace Kentico.Kontent.Management.Tests.Unit.Modules.UrlBuilder
```

Comment:
I think we should separate tested method in test name with '_'. It might improve readability.

Reply:
Can we create a bug for renaming all asset + assetRendition tests in `EndpointUrlBuilderTests` class? I guess you are right that we are not consistent with naming and we should be. On the other hand it's out of the scope for this PR.